kythe/go/serving/explore/explore_test.go

```diff
@@ -0,0 +1,478 @@
+package explore
```

Comment:
Run `arc lint`.  You're missing copyright headers.

Reply:
`arc lint` doesn't seem to do anything.  Added copyright header.